src/app/complex/complex-results/complex-navigator/table-structure/table-structure.component.ts

```diff
@@ -20,7 +20,7 @@ export class TableStructureComponent implements OnChanges {
 
   ngOnChanges(): void {
     this.classifyComplexesSize(); // ensure to always have the same base of comparison
-    this.sortedComplexes = this.classifyComplexesSimilarities(this.complexSearch.elements);
+    this.sortedComplexes = this.classifyComplexesSimilaritiesV2(this.complexSearch.elements);
```

Comment:
What has changed here?

Reply:
the classifyComplexesSimilaritiesV2 is the function supposed to cluster the complexes but it is not finished yet, right now it is only classifying the complexes depending on the similarity score they obtain